magda-web-client/src/Components/Dataset/Add/DatasetAddMetadataPage.tsx

```diff
@@ -256,7 +256,7 @@ class NewDataset extends React.Component<Props, State> {
                     <div>
                         <MultilineTextEditor
                             value={datasetPublishing.notesToApprover}
-                            placerHolder="Enter additional notes"
+                            placeHolder="Enter additional notes"
```

Comment:
@sajidanower23 
Good pickup on the typo~ 👍 
Since you're already here, could you please make the spelling more consistent with actual DOM element property name?
i.e. Change it all `placeHolder` to `placeholder`

Reply:
Sure thing. Done.